DFE-Digital/early-careers-framework
# CPDLP-267 Add payable view to payment breakdown

## Ticket and context

Ticket: https://dfedigital.atlassian.net/jira/software/projects/CPDLP/boards/87?assignee=5e4e50b03df51b0c9374c1e4&selectedIssue=CPDLP-267

- Added LHS nav when view in payment breakdown
- On top of the current view there is now a payable view for those items in `payable` state

## Screenshots

![Screenshot 2021-11-04 at 12 48 39](https://user-images.githubusercontent.com/92580/140316169-34f7b07b-0aa7-481a-8088-06681d8a547b.png)

## Tech review

### Is there anything that the code reviewer should know?

### Code quality checks

### HTML Checks

### Gotchas

## Product review

### How can someone see it it review app?

- Visit https://ecf-review-pr-1397.london.cloudapps.digital
- Login as `finance@example.com`
- View ECF payment breakdowns
- Should see current and payable views
- Unfortunately insufficient data has been setup to differentiate the data between the views

## External API changes

- There are no external API changes
